mmv1/third_party/terraform/website/docs/guides/using_write_only_arguements.html.markdown

```diff
@@ -1,29 +1,29 @@
 ---
-page_title: "Use write-only attributes in the Google Cloud provider"
+page_title: "Use write-only arguments in the Google Cloud provider"
 description: |-
-  How to use write-only attributes in the Google Cloud provider
+  How to use write-only arguments in the Google Cloud provider
 ---
 
-# Write-only attributes in the Google Cloud provider
+# Write-only arguments in the Google Cloud provider
 
-The Google Cloud provider has introduced new write-only attributes for a more secure way to manage data. The new `WriteOnly` attribute accepts values from configuration and will not be stored in plan or state providing an additional layer of security and control over data access.
+The Google Cloud provider has introduced new write-only arguments for a more secure way to manage data. The new `WriteOnly` attribute accepts values from configuration and will not be stored in plan or state providing an additional layer of security and control over data access.
 
-For more information, see the [official HashiCorp documentation for Write-only Attributes](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
+For more information, see the [official HashiCorp documentation for Write-only Arguments](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/write-only-arguments).
 
-The Google Cloud provider has added the following write-only attributes:
+The Google Cloud provider has added the following write-only arguments:
```

Comment:
This list is out of date. We probably should delete it / restructure this... but that can happen separately.